app/controllers/users_controller.rb

```diff
@@ -51,7 +51,7 @@ def reorder_call_list
   private
 
   def user_params
-    params.require(:user).permit(:name, :email, :_role)
+    params.require(:user).permit(:name, :email, :role)
```

Comment:
I think that you can probably provide any scripts for you know, migrating the data as a part of this PR rather than a separate one, unless you feel absolutely that you need to break it apart.